docs/source/contributing.rst

```diff
@@ -27,7 +27,7 @@ Contributions to Receptor go through the Github pull request process.
 An initial checklist for your change to increase the likelihood of acceptance:
 - No issues when running linters/code checkers
 - No issues from unit/functional tests
-- Write good commit messages. See [How to write a Git commit message](https://cbea.ms/git-commit/).
+- Write good commit messages. See How to write a Git commit message<https://cbea.ms/git-commit/>.
```

Comment:
```suggestion
- Write good commit messages. See `How to write a Git commit message<https://cbea.ms/git-commit/>`_.
```

Reply:
I would also qualify what we mean by good commit messages so it's not so subjective. Perhaps something like: "Write descriptive and meaningful commit messages."

I also like this resource (which also refers to "good commit messages" lol): https://gist.github.com/webknjaz/cb7d7bf62c3dda4b1342d639d0e78d79

Reply:
Thank you for the clarifications.  I've added them in the next set of commits